src/kernel/src/arch/amd64/interrupt.rs

```diff
@@ -400,6 +400,7 @@ fn generic_isr_handler(ctx: *mut IsrContext, number: u64, user: bool) {
         );
     }
 
+    lapic::eoi();
     match number as u32 {
         14 => {
```

Comment:
What is interrupt 14?

Reply:
Page-fault. I'll clean up these old magic numbers in the arch code in a separate PR to avoid cluttering this one.